<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-IHMLS1RS" failures="1" tests="1" name="td.Runner.FailedRun" time="0.085" errors="0" timestamp="30 Jun 2023 15:44:27 GMT" skipped="0">
  <testcase classname="td.Runner.FailedRun" name="runScenario" time="0.085">
    <failure message="expected [TDBankClosed] but found [TDClose]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [TDBankClosed] but found [TDClose]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at td.StepDefinitions.CarQuoteSteps.open_a_web_browser(CarQuoteSteps.java:11)
at âœ½.open a web browser(file:///C:/Users/harih/eclipse-workspace/TDCucumberFramework/src/test/java/td/Features/tdcarquote.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- td.Runner.FailedRun -->
